nncf/experimental/quantization/compression_builder.py

```diff
@@ -78,6 +78,11 @@ def apply(self, model: TModel, dataset: Dataset) -> TModel:
         5) Collect all statistics.
         6) Apply algorithms.
         """
+        NNCFTelemetry.start_session(NNCF_ONNX_CATEGORY)
+        algo_names = sorted([x.__class__.__name__ for x in self.algorithms])
+        NNCFTelemetry.send_event(event_category=NNCF_ONNX_CATEGORY,
+                                 event_action='compression_started',
+                                 event_label=','.join(algo_names))
```

Comment:
Do we also need to send the algorithm configurations used for quantization for further analysis?